### DISCUSSION TITLE
Container keeps hibernating even with clients connected

### DISCUSSION THREAD

**Discussion by tritone246** [codesandbox/codesandbox-client#4797] [General] [answered]
I'm coding a multiplayer game, but my codesandbox container keeps hibernating, even when clients are connected to the server.
Basically, my game is unplayable since it resets every 15 minutes.
Does anyone know why this is happening?
thanks

**Comment by SaraVieira** [comment, marked as answer]
Hey!

This was an issue with had where we weren't waking up containers. I think this was fixed yesterday

Is it still happening?

**Reply by tritone246** [reply]
I don't think so. Thank you!!!!!

**Reply by tritone246** [reply]
Oops, this is happening again. I think it is because I never create a HTTP server object, I'm only using websockets and express.... but shouldn't it still detect if people are on the project page and thus not close the server? idk

**Reply by SaraVieira** [reply]
Let me ping @lbogdan

**Reply by lbogdan** [reply]
Hey @haha0201 ,

We only detect sandbox / container activity when loading a html page. Please note that CodeSandbox is primarily (at least for now) a development platform; if you want to deploy your project you can use a hosting platform such as [Vercel or Netlify](https://codesandbox.io/docs/deployment). That being said, we're currently evaluating the option of providing always-on container sandboxes.

**Reply by tritone246** [reply]
So basically, you only detect if a person goes ahead and opens a new tab, not if a person is already on the page?

**Reply by lbogdan** [reply]
Yes, that's correct.

**Reply by tritone246** [reply]
Alright, when will this be fixed?
~haha0201

**Reply by lbogdan** [reply]
As I said above, this is expected behavior, so there's nothing to fix.

If you're asking about the always-on containers feature, we'll most probably release it until the end of this month.
